src/test/query-user.test.ts

```diff
@@ -3,13 +3,22 @@ import { expect } from 'chai';
 import { cleanAll } from './clear';
 import Jwt from 'jsonwebtoken';
 import { createUser } from './create-user';
+import { createJwtToken } from './createJwtToken';
+import { User } from '../entity/User';
+
+describe.only('Graphql - Query User', () => {
```

Comment:
atenção com esses `only`
```suggestion
describe('Graphql - Query User', () => {
```